hack/make-rules/verify.sh

```diff
@@ -38,6 +38,7 @@ QUICK_PATTERNS+=(
   "verify-boilerplate.sh"
   "verify-godep-licenses.sh"
   "verify-gofmt.sh"
+  "verify-multiarch-test-images.sh"
```

Comment:
Can we rename this to just `verify-test-images.sh`?
I think that's more correct, as we can generally enforce the layout of that directory, not just that it's multiarch...